auxx/src/Command/Update.hs

```diff
@@ -79,20 +78,19 @@ propose sendActions ProposeUpdateParams{..} = do
     CmdCtx{ccPeers} <- getCmdCtx
     logDebug "Proposing update..."
     skey <- (!! puIdx) <$> getSecretKeys
-    let BlockVersionData {..} = genesisBlockVersionData
     let bvm =
             BlockVersionModifier
-            { bvmScriptVersion     = puScriptVersion
-            , bvmSlotDuration      = convertUnit (sec puSlotDurationSec)
-            , bvmMaxBlockSize      = puMaxBlockSize
-            , bvmMaxHeaderSize     = bvdMaxHeaderSize
-            , bvmMaxTxSize         = bvdMaxTxSize
-            , bvmMaxProposalSize   = bvdMaxProposalSize
-            , bvmMpcThd            = bvdMpcThd
-            , bvmHeavyDelThd       = bvdHeavyDelThd
-            , bvmUpdateVoteThd     = bvdUpdateVoteThd
-            , bvmUpdateProposalThd = bvdUpdateProposalThd
-            , bvmUpdateImplicit    = bvdUpdateImplicit
+            { bvmScriptVersion     = Just puScriptVersion
+            , bvmSlotDuration      = Just $ convertUnit (sec puSlotDurationSec)
+            , bvmMaxBlockSize      = Just puMaxBlockSize
+            , bvmMaxHeaderSize     = Nothing
+            , bvmMaxTxSize         = Nothing
+            , bvmMaxProposalSize   = Nothing
+            , bvmMpcThd            = Nothing
+            , bvmHeavyDelThd       = Nothing
+            , bvmUpdateVoteThd     = Nothing
+            , bvmUpdateProposalThd = Nothing
+            , bvmUpdateImplicit    = Nothing
```

Comment:
Ah, probably it would be better to define a `BlockVersionModifier` with all `Nothing`s somewhere in `cardano-sl` and only set relevant fields here.
Anyway, not really important.